api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -457,6 +461,128 @@ def validate_finance_incident(finance_incident_id: int) -> utils.BackofficeRespo
     return render_finance_incident(finance_incident)
 
 
+@finance_incidents_blueprint.route("/<int:finance_incident_id>/force-debit-note", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def get_finance_incident_force_debit_note_form(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = finance_models.FinanceIncident.query.filter_by(id=finance_incident_id).one_or_none()
+
+    if not finance_incident:
+        raise NotFound()
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=empty_forms.EmptyForm(),
+        dst=url_for("backoffice_web.finance_incidents.force_debit_note", finance_incident_id=finance_incident_id),
+        div_id=f"finance-incident-force-debit-note-modal-{finance_incident_id}",
+        title="Générer une note de débit",
+        button_text="Confirmer",
+        information='Vous allez choisir le mode de compensation "Note de débit", une note de débit sera envoyée à l\'acteur culturel à la prochaine échéance. Voulez-vous continuer ?',
+    )
+
+
+@finance_incidents_blueprint.route("/<int:finance_incident_id>/force-debit-note", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def force_debit_note(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = _get_incident(finance_incident_id)
+
+    if finance_incident.status != finance_models.IncidentStatus.VALIDATED:
+        flash("L'incident doit être au statut 'validé' pour effectuer cette action.", "warning")
+        return redirect(
+            request.referrer
+            or url_for("backoffice_web.finance_incidents.get_incident", finance_incident_id=finance_incident.id),
+            303,
+        )
+
+    if finance_incident.isClosed:
+        flash("Impossible d'effectuer cette action sur un incident terminé.")
+        return redirect(
+            request.referrer
+            or url_for("backoffice_web.finance_incidents.get_incident", finance_incident_id=finance_incident.id),
+            303,
+        )
+
+    finance_incident.forceDebitNote = True
+    db.session.add(finance_incident)
+
+    action = history_api.log_action(
+        history_models.ActionType.FINANCE_INCIDENT_DEBIT_NOTE_CHOICE,
+        author=current_user,
+        finance_incident=finance_incident,
+        comment="Le mode de compensation est passé à 'Note de débit'.",
```

Comment:
Je ne suis pas partisan d'un commentaire rédigé dans ce cas, mais plutôt d'un chanp en `extraData` qui donne l'info, la mise en forme étant faite au moment du rendu. Donc ici ce serait du genre : 
```suggestion
        force_debit_note=True,
```
et à l'inverse, `False` dans le cas de la fonction suivante.

Reply:
J'ai du mal à voir l'attendu ici, l'ajout de cette donnée ne semble pas affecter l'affichage (j'ai simplement un commentaire vide)

Reply:
Il faut ajouter quelque chose là-dedans pour afficher l'info : 
https://github.com/pass-culture/pass-culture-main/blob/v270.0.0/api/src/pcapi/routes/backoffice/templates/components/history/actions.html#L38

Reply:
Un truc comme ca ? 
![image](https://github.com/pass-culture/pass-culture-main/assets/134523772/ef31c76a-57a3-442e-81dd-5903326f50ff)

Reply:
Je pensais à un traitement spécifique dans le jinja mais en effet ta proposition marche aussi, en stockant dans extraData["modified_info"].